geode-redis/src/distributedTest/java/org/apache/geode/redis/executors/keys/PersistDUnitTest.java

```diff
@@ -134,10 +133,8 @@ public void testConcurrentPersistOperations() throws InterruptedException {
 
   private void setKeysWithExpiration(Jedis jedis, Long iterationCount, String key) {
     for (int i = 0; i < iterationCount; i++) {
-      SetParams setParams = new SetParams();
-      setParams.ex(600);
-
-      jedis.set(key + i, "value" + i, setParams);
+      jedis.sadd(key + i, "value" + 9);
+      jedis.expire(key + i, 600);
```

Comment:
Is there a resion why we're calling expire separately instead of using a set option?

Reply:
early on this branch, the new expiration was only working on sets and hashes. So I changed these tests to use sets instead of strings. It seems like this test is okay with any data type so I left it using a set.